### PR TITLE
Add problems pane to show warnings

### PR DESCRIPTION
**Pull request by remcohaszing**
This has been copy-pasted from monaco-yaml.

### CODE REVIEW

File: examples/demo/src/index.ts [hunk 1]
```diff
@@ -186,8 +186,40 @@ const ed = editor.create(document.getElementById('editor')!, {
   model: getModel(),
 });
 
+function updateMarkers(resource: Uri): void {
```
Comment (mhsdesign) [inline examples/demo/src/index.ts:189]:
```suggestion
// totally optional functionality - and not tailwind specific
function updateMarkers(resource: Uri): void {
```

**Review by mhsdesign — COMMENTED** [review]
Nice and cool! Love that this works out of the box without further changes :D
Since this is more of an optional thing and i think we maybe should make that somehow clear as otherwise it would be getting harder and harder to 'destructure' the demo into an implementation that one needs. (Id like to do this often, grap a demo and start 'trashing' the stuff one doesnt need.)

And I must say that i love that the currently implementation is so super simple ^^
But then again, even more simple can be found in the esbuild demo so then we can fit all the fancy features in this one and show of i guess?

**Comment by remcohaszing** [comment]
> And I must say that i love that the currently implementation is so super simple ^^

I agree with that!

That having said, I do like this problems pane. I initially excluded it, because I didn’t realize the language service provides diagnostics.

I like how I did this in `monaco-yaml`: https://github.com/remcohaszing/monaco-yaml/tree/main/examples. There are some very minimal examples, and there’s the demo that does just a little more.

**Comment by mhsdesign** [comment]
Then let's merge it ;)

**Pull request merged**